### PR TITLE
Issue 38

### PR DESCRIPTION
**Pull request by iamalexk**
Fixed get_standings function from scraper.py and updates predictions.html to automatically update standings

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -93,7 +93,10 @@ def authors():
 def predictions():
     bracket = get_playoff_bracket()
     bracket = create_html_bracket(bracket)
-    return render_template("predictions.html", bracket=bracket)
+    west_standings = get_standings("west")
+    east_standings = get_standings("east")
+    print(east_standings)
```
Comment (skekre98) [inline app.py:98]: Let's remove this print statement

File: modules/scraper.py [hunk 2]
```diff
@@ -133,19 +133,24 @@ def get_standings(conf):
     resp = requests.get(base_url)
     page_content = BeautifulSoup(resp.content, "html.parser")
     table = page_content.findAll("tr",attrs={"class":"full_table"})
-
+    
     # Iterate over ranking table in website HTML 
     standings = []
-    for i,row in enumerate(table):
-        team = row.find("a")["title"]
+    # print(table[0])
```
Comment (skekre98) [inline modules/scraper.py:139]: Let's remove these extra comments

**Review by skekre98 — COMMENTED** [review]
@iamalexk looks great! I have added a couple review comments. Can you also retrain the query classifier, it seems it was accidentally removed in one of your commits?

**Comment by iamalexk** [comment]
@skekre98 I've fixed your comments, but what do you mean by retraining the query classifier?

**Comment by skekre98** [comment]
> @skekre98 I've fixed your comments, but what do you mean by retraining the query classifier?

@iamalexk if you look at the merge conflict, you'll see that git is complaining because `query_classifier.pkl` has been deleted. Not sure how this happened, but you can fix this by running `python query_classifier.py`

**Comment by iamalexk** [comment]
@skekre98 Can you review my pull request. If query_classifier.pkl is still missing will you let me know. It looks to be in inference/models so I dont know why it would be causing a merge conflict. Thanks!

**Comment by iamalexk** [comment]
@skekre98 Not sure what's going on with query_classifier.pkl. I added query_classifier.pkl in my commit but it still seems to have a conflicting file issue. Is there any way for you to merge just the app.py file and the modules/scraper.py files? I shouldn't have committed any of the changes I made to the other files I modified.

File: query_classifier.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 # ML packages
 from sklearn.svm import LinearSVC
 from sklearn.pipeline import Pipeline
-from sklearn.externals import joblib
```
Comment (skekre98) [inline query_classifier.py:9]: Let's change this back to import `joblib` from `sklearn.externals`.

File: inference/inference_network.py [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import pickle
 from sklearn.externals import joblib
+import joblib
```
Comment (skekre98) [inline inference/inference_network.py:3]: Let's remove this extra joblib import.

**Comment by skekre98** [comment]
I think the issue is being created because you are using a different version of joblib and I'm assuming these versions encode their pickle files differently. Let's change this back to the original version of `joblib` that we were importing from `sklearn`.

**Comment by skekre98** [comment]
The brute force solution is to save the changes you've made to `app.py`, `scraper.py`, and `predictions.html` and `rebase` with the your fork. You can then re-add your changes and this will definitely resolve the merge conflict.

**Comment by skekre98** [comment]
> @skekre98 Not sure what's going on with query_classifier.pkl. I added query_classifier.pkl in my commit but it still seems to have a conflicting file issue. Is there any way for you to merge just the app.py file and the modules/scraper.py files? I shouldn't have committed any of the changes I made to the other files I modified.

@iamalexk Unfortunately I can only merge a pull request, not files, you will have to remove the unnecessary changes from your PR. Refer to this stack-overflow query about how to remove files from a PR: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

Keep in mind, deleting a file and removing a file from a PR are two separate things.

**Comment by iamalexk** [comment]
@skekre98 I have issued a new pull request called new-issue-38 that should hopefully fix our problems. Can we refer to that pull request from here?

**Pull request closed without merging**